### PR TITLE
Mia pt Manuel cabrera

### PR DESCRIPTION
**Pull request by ManuelCabrera01**

### CODE REVIEW

File: starter-code/iteration2/iteration2/src/app/app.module.ts [hunk 1]
```diff
@@ -1,18 +1,20 @@
...
+    MyIteration2Component,
+    MyIteration3Component,
+
   ],
   imports: [
-    BrowserModule,
-    FormsModule,
-    HttpModule
+    BrowserModule
+      FormsModule
```
Comment (sandrabosk) [inline starter-code/iteration2/iteration2/src/app/app.module.ts]: You are missing semicolon after BrowserModule - that's crashing your app.

**Comment by sandrabosk** [comment]
In the second iteration we have some article that has title, image and text and after that we have some comments (each comment has author, content and time). The last thing in this iteration is the form for new comments. Following the previous iteration and class examples, make it using Angular2 syntax. If you need help, please reach out to me. It is extremely important that you understand and know how to apply basic concepts in this module.

**Comment by sandrabosk** [comment]
I am going to be closing the pool requests on Saturday, Dec 9th, so by then please submit the rest of the exercise. Thank you.

**Comment by ManuelCabrera01** [comment]
So this means that I'm officially done with my homework😀

**Comment by ManuelCabrera01** [comment]
?

**Comment by sandrabosk** [comment]
Hahahahha, true 😊😊😊

________________________________

**Pull request closed without merging**